source/faucet/main.d

```diff
@@ -266,10 +316,12 @@ public class Faucet : FaucetAPI
 
     ***************************************************************************/
 
-    public this (const Config config, const Address address)
+     public this (in Config config)
```

Comment:
Unnecessary space.

Reply:
Also it's being stored in the class so by definition it is not `scope`.

Reply:
Made it global.